### DISCUSSION TITLE
Welcome to CostBot Discussions!

### DISCUSSION THREAD

**Discussion by Costpap** [Costpap/CostBot#142] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlmostSuspense** [comment]
!rank

**Reply by Costpap** [reply]
Unfortunately, CostBot does not have a ranking system nor is it currently planned.

**Reply by hmellow** [reply]
@Mee6 help

**Reply by AlmostSuspense** [reply]
sorry i thought it was mee6 support

**Reply by hmellow** [reply]
It is

**Reply by Costpap** [reply]
> @Mee6 help

Did you actually ping them, lmao

**Reply by Costpap** [reply]
> sorry i thought it was mee6 support

It's fine.

**Reply by AlmostSuspense** [reply]
!rank

**Reply by AlmostSuspense** [reply]
@costpag it still doesnt work tho

**Reply by Costpap** [reply]
fuck you

**Reply by hmellow** [reply]
@Costpap would you pls fizx the mee6 bot I want rto use anusm e comands

**Reply by hmellow** [reply]
!rank

**Reply by hmellow** [reply]
!rank

**Reply by hmellow** [reply]
!help

**Reply by Costpap** [reply]
I cannot offer any help with MEE6, unfortunately.

**Comment by hmellow** [comment]
Costbot is my favorite ~~anime~~ bot

Edited by @Costpap: CostBot is not an anime bot and there are no plans at this time to make it into one.

**Reply by AlmostSuspense** [reply]
help someone is abusing their edit perms

**Reply by hmellow** [reply]
I just wish the anime features weren't locked to the biggest anime fan, costpap. I hear he's making his own Greek anime series in 2022!

**Reply by Costpap** [reply]
I do not see any evidence of abuse.

**Reply by Costpap** [reply]
CostBot does not have any anime features at this moment. I never plan on making a Greek anime bot in 2022.

**Reply by AlmostSuspense** [reply]
how do i edit other people messages

**Reply by Costpap** [reply]
You can become a maintainer in order to do that. However, it is extremely unclear, even to me, how that can happen.

**Reply by hmellow** [reply]
> CostBot does not have any anime features at this moment. I never plan on making a Greek anime bot in 2022.

2023 icic

**Reply by Costpap** [reply]
That is untrue.

**Reply by hmellow** [reply]
Why did you open a discussion for us smh

**Reply by Costpap** [reply]
🤷
